roles/openshift_master/tasks/main.yml

```diff
@@ -258,6 +258,23 @@
   retries: 1
   delay: 60
 
+- name: configure vsphere svc account
```

Comment:
Why is this part of the master role?  Shouldn't all cloud provider config be done from the openshift_cloud_provider role and rely on cloud provider auto detection?  I don't think this belongs here.